From 5a1d14de75a4d7067c3f924b20f3092c9abdc8c2 Mon Sep 17 00:00:00 2001
From: Kenny Kessler <kenny.kessler@motorola.com>
Date: Tue, 10 Nov 2015 10:46:06 -0600
Subject: [PATCH 466/959] IKSWM-8388: arm/dt: msm8952-moto: Refactor Charge
 Voltage

Move the PMI8952 Charge Votlage setting back into the
common file since all batteries will be 4.35V based.

Change-Id: I7a94582e40716bea50e39a8ac5e4a6513f2cb00e
Signed-off-by: Kenny Kessler <kenny.kessler@motorola.com>
Reviewed-on: http://gerrit.mot.com/798727
SME-Granted: SME Approvals Granted
SLTApproved: Slta Waiver <sltawvr@motorola.com>
Tested-by: Jira Key <jirakey@motorola.com>
Reviewed-by: Jason Knopsnyder <jasonk@motorola.com>
Submit-Approved: Jira Key <jirakey@motorola.com>
---
 arch/arm/boot/dts/qcom/msm8952-athene-p0.dts    |  8 --------
 arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi | 10 ++++++++--
 2 files changed, 8 insertions(+), 10 deletions(-)

diff --git a/arch/arm/boot/dts/qcom/msm8952-athene-p0.dts b/arch/arm/boot/dts/qcom/msm8952-athene-p0.dts
index 0cd9ce3ab89..520e2a94811 100644
--- a/arch/arm/boot/dts/qcom/msm8952-athene-p0.dts
+++ b/arch/arm/boot/dts/qcom/msm8952-athene-p0.dts
@@ -91,14 +91,6 @@
 	};
 };
 
-&pmi8950_charger {
-	qcom,float-voltage-mv = <4350>;
-};
-
-&pmi8950_fg {
-	qcom,fg-cc-cv-threshold-mv = <4340>;
-};
-
 &i2c_2 {
 	cs35l34@40 {
 		status = "disabled";
diff --git a/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi b/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi
index 7caa3f6a9fe..066311b4622 100644
--- a/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi
+++ b/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi
@@ -293,7 +293,7 @@
 	qcom,usbid-disabled;
 	/delete-property/ qcom,autoadjust-vfloat;
 	qcom,fastchg-current-ma = <2500>;
-	qcom,float-voltage-mv = <4400>;
+	qcom,float-voltage-mv = <4350>;
 	qcom,resume-delta-mv = <200>;
 	qcom,chg-inhibit-fg;
 	qcom,rparasitic-uohm = <100000>;
@@ -335,6 +335,12 @@
 	};
 };
 
+&mot_batterydata {
+	qcom,fx30-lg {
+		qcom,max-voltage-uv = <435000>;
+	};
+};
+
 &pmi8950_fg {
 	status = "okay";
 	qcom,warm-bat-decidegc = <450>;
@@ -353,7 +359,7 @@
 	qcom,fg-delta-soc = <1>;
 	qcom,fg-max-soc = <100>;
 	qcom,fg-min-soc = <0>;
-	qcom,fg-cc-cv-threshold-mv = <4390>;
+	qcom,fg-cc-cv-threshold-mv = <4340>;
 	qcom,battery-data = <&mot_batterydata>;
 };
 
-- 
2.11.0

